CHANGES.md

```diff
@@ -18,6 +18,7 @@
 - Report `#require` directive errors (#276, @gpetiot)
 - Handle no such file exception: the input file and the values of options `--root` and `--prelude` are checked (#292, @gpetiot)
 - Keep locations from parsing instead of recomputing the lines, providing better error messages (#241, @gpetiot)
+- Use with-accepted-exit-codes in tests instead of system (#301 @tmattio)
```

Comment:
I'm sorry that @emillon had you fix this but we only add user facing changes to the changelog and I don't think changes in our test suite belong here.

Reply:
That makes sense! The Github build will fail without the changelog entry, but if that's ok I'll just remove it

Reply:
if you set the `no-changelog-needed` label (or let @NathanReb do it) it'll skip the check.

Reply:
I don't seem to be able to manage labels